### DISCUSSION TITLE
SC as Euphonium Plugin?

### DISCUSSION THREAD

**Discussion by AxelJust** [CarlosDerSeher/snapclient#28] [Ideas]
Is it realistic to implement it as [Euphonium](https://github.com/feelfreelinux/bell) Plugin?

**Comment by CarlosDerSeher** [comment]
I don't really know the internals of Euphonium so can't give a good answer here. It mostly depends on how much abstraction there is in the playback part. You really need close control of the i2s part. 
Maybe a short example, if there is a ring buffer in Euphonium you push received samples in and the playback is handled elsewhere then no, it won't be possible easily.
